src/io/libxdrf.c

```diff
@@ -0,0 +1,1008 @@
+/*
```

Comment:
`write_xtc`, `read_xtc` functions are available in libgromacs.so so maybe it is possible to use them instead of attaching this file? They are declared here: `gromacs/fileio/xtcio.h`; not sure how difficult is to use it from outside.
With this, we will not have a problem of migrating any changes from GROMACS base code to our version of this library.
We could also add compilation flag to switch on this module which will set requirements on GROMACS..